src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -377,4 +387,21 @@ public without sharing class UTIL_CustomSettingsFacade {
         }
         return false;
     }
+
+    /*******************************************************************************************************
+    * @description Executes Ethnicity Race Batch
+    * @return void
+    */
+    @AuraEnabled
+    public static Id executeEthnicityRaceBatch() {
```

Comment:
This method doesn't really belong here. I know we talked about moving these controller methods to a separate class. Let's make sure to move this one out too when that happens.